### PR TITLE
Added PR template

### PR DESCRIPTION
**Pull request by MarkWareham**

### CODE REVIEW

**Comment by MarkWareham** [comment]
Jenkins test please

**Comment by Pandrex247** [comment]
Is there a way to properly review this without merging it first?

**Comment by arjantijms** [comment]
The points looks good, not sure if it "works" as PR template.

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by MarkWareham** [comment]
> shouldn't this be in .GitHub? Also isn't the name incorrect for a pull request template.
> 
> I would put optional stuff at the bottom of the template not the top.

Optional to put in .github. 
Name is correct according to the help: https://help.github.com/en/articles/creating-a-pull-request-template-for-your-repository.
I put the optional stuff at the top as it's important stuff. If there's any blockers or deadlines we want to see them easily/not miss them

**Comment by MarkWareham** [comment]
@Pandrex247 wrote
> Is there a way to properly review this without merging it first?
The content can be reviewed. Whether it works is different, see below

@arjantijms  wrote:
> The points looks good, not sure if it "works" as PR template.
I tested in my own repo. Template needs to be in master to work:
image

**Comment by jbee** [comment]
This PR does not follow the PR template :D

File: pull-request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#(Optional) : Deadline for Merge
+
+Is there a deadline other than sprint end for the merge to happen?
```
Comment (Pandrex247) [inline pull-request_template.md]: Do we have deadlines that aren't sprint end frequently enough to warrant this?

**Comment by MarkWareham** [comment]
Hadn't read the final comments before hitting merge. However they did not explicitly request changes. I'll create another PR to tweak

**Comment by MarkWareham** [comment]
> This PR does not follow the PR template :D

What did you mean by this @jbee ?

**Pull request merged**